config/webpack.common.config.js

```diff
@@ -7,6 +7,8 @@ module.exports = {
     assets: path.resolve(__dirname, '../src/index.jsx'),
     accessibilityPolicy: path.resolve(__dirname, '../src/accessibilityIndex.jsx'),
     editImageModal: path.resolve(__dirname, '../src/editImageModalIndex.jsx'),
+    courseHealthCheck: path.resolve(__dirname, '../src/courseHealthCheckIndex.jsx'),
+    courseOutlineHealthCheck: path.resolve(__dirname, '../src/courseOutlineHealthCheckIndex.jsx'),
```

Comment:
No need to do this now, but we should probably synchronize this and the other webpack config to the Checklist naming in the next PR.